python/adelphi/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+setup(
+   name='adelphi',
+   version='0.1.0',
+   author='Gianluca Righetto',
+   author_email='gianluca.righetto@datastax.com',
+   packages=['adelphi'],
+   scripts=['bin/adelphi'],
+   #url='http://pypi.python.org/pypi/PackageName/',
+   url='https://github.com/datastax/adelphi',
+   #license='LICENSE.txt',
+   description='Schema anonymizer for Adelphi',
+   #long_description=open('README.txt').read(),
+   install_requires=['cassandra-driver', 'click'],
```

Comment:
it's probably a good idea to pin the cassandra-driver to a specific version such that it doesn't update under our feet as it could export more schema information in future versions

Reply:
Agreed, I'll fix this.